build.sbt

```diff
@@ -70,5 +70,15 @@ lazy val tests = crossProject(JVMPlatform, JSPlatform)
     skip in publish := true
   )
   .jvmConfigure(_.enablePlugins(AutomateHeaderPlugin))
+  .jvmSettings(
+    resolvers += "Gemini Repository" at "https://github.com/gemini-hlsw/maven-repo/raw/master/releases",
+    libraryDependencies ++= Seq(
+      "edu.gemini.ocs" %%% "edu-gemini-util-skycalc" % "2020001.1.7",
```

Comment:
I suppose we don't need three `%` for the jvm side

Reply:
Good catch. Fixed.